test/raft/randomized_nemesis_test.cc

```diff
@@ -3169,8 +3169,10 @@ SEASTAR_TEST_CASE(basic_generator_test) {
             ? raft::server::configuration {
                 .snapshot_threshold{10},
                 .snapshot_trailing{5},
-                .max_log_size{20},
+                .max_snapshot_trailing_bytes{150},
+                .max_log_size{450},
                 .enable_forwarding{forwarding},
+                .max_command_size{150},
```

Comment:
How did you choose those numbers?

Reply:
The command size is no more than 150 (the commands themselves are small here + `sizeof(log_entry) == 136`) , we want frequent snapshots and the [condition](https://github.com/scylladb/scylladb/pull/11397/files#diff-653cd62bbe7be7fd70f5ef5165df3b66280bdd53c23a57bd4d9c79412f071c60R47) should be me.

It is worth adding this text as a comment here.

Reply:
May be we can express it here based on the max command size? Is there a constant that defines it?

Reply:
done